source/includes/_api.md

```diff
@@ -341,3 +341,162 @@ The example to the right shows only how to fetch information for the courses, ac
+  --compressed 
+```
+
+As an agency customer, you can retrieve data on any of the invoices that have been created through Edvisor. Please
+see the example on the right. You can structure the query to get additional info on the invoice, such as any files
+associated with it, any invoice items (referred to as raw items), the invoice type, and the payment currency.
+
+There are two different endpoints to query invoices, either by Invoice Id or using a filtered search. 
+
+The following parameters are available for the filtered search:
+<ul>
+  <li>agencyIds: number []</li>
+  <li>invoiceIds: number []</li>
+  <li>externalIds: number []</li>
+  <li>created: DateOnlyRangeInput</li>
+</ul>
+
+This query also takes a pagination input, which is an object with an offset and a limit field.
+
+The example on the right shows a filtered search using agencyIds and the created parameter.
+
+The created parameter can take the following filters:
+<ul>
+  <li>gte: greater than or equal</li>
+  <li>gt: greater than</li>
+  <li>lte: less than or equal</li>
+  <li>lt: less than</li>
+  <li>eq: equal</li>
+</ul>
+
+```
+query {
+  invoiceList(
+  pagination: { limit: 100, offset:0 }, 
+  filter: {
+    agencyIds: [1, 2],
+    created: {
+      gt: "2018-01-31"
+    }
+  }) {
+    count
+    data {
+      ...
+    } 
+  }
+    
+}
+```
+
+For more details as to which fields are available on which objects, consult the schema documentation. 
+
+* <a href='http://docs.edvisor.io/schema/invoice.doc.html'>Invoice</a>
+* <a href='http://docs.edvisor.io/schema/invoiceItem.doc.html'>InvoiceItem</a>
```

Comment:
i think these need to be lowercase... for example `http://docs.edvisor.io/schema/invoiceItem.doc.html` should be `http://docs.edvisor.io/schema/invoiceitem.doc.html`